backend/clap/build.gradle

```diff
@@ -29,8 +30,10 @@ dependencies {
             "org.springframework.security:spring-security-oauth2-jose",
             "org.springframework.security:spring-security-oauth2-resource-server",
             "org.springframework.boot:spring-boot-starter-validation",
-            "com.github.ben-manes.caffeine:caffeine:3.1.8"
+            "com.github.ben-manes.caffeine:caffeine:3.1.8",
+            "org.springframework.boot:spring-boot-starter-webflux"
     )
+    testImplementation 'org.testng:testng:7.1.0'
```

Comment:
There is already `testImplementation` block few lines below. Can we put it there?

Reply:
good idea, I didn't see it.